tests/models/test_dag.py

```diff
@@ -822,6 +822,36 @@ def test_bulk_write_to_db_max_active_runs(self, state):
         model = session.query(DagModel).get((dag.dag_id,))
         assert model.next_dagrun_create_after is None
 
+    def test_bulk_write_to_db_has_import_error(self):
+        """
+        Test that DagModel.has_import_error is set to false if no import errors.
+        """
+        dag = DAG(dag_id='test_has_import_error', start_date=DEFAULT_DATE)
+
+        DummyOperator(task_id='dummy', dag=dag, owner='airflow')
+
+        session = settings.Session()
```

Comment:
```suggestion
    def test_bulk_write_to_db_has_import_error(self, session):
        """
        Test that DagModel.has_import_error is set to false if no import errors.
        """
        dag = DAG(dag_id='test_has_import_error', start_date=DEFAULT_DATE)

        DummyOperator(task_id='dummy', dag=dag, owner='airflow')

```

Use the session fixture?

Reply:
Ah, yeah. That's better

Reply:
Turns out the class TestDag is using unittest.TestCase and not pytest. Wondering if it's worth it to convert it to pytest in this PR so we can use the session fixture or create a session fixture inside the unittest class as shown here?: https://pytest.org/en/latest/how-to/unittest.html#using-autouse-fixtures-and-accessing-other-fixtures

Reply:
I'd say do that in another PR. Plenty of other tests could benefit from it as well.